.gitignore

```diff
@@ -20,13 +21,32 @@ build-*
 *.cfg
 
 # Game data
-csm.bin
-CSM.BIN
-CSM.bin
-csm.BIN
+*.bin
+*.BIN
+CURSED/
+CURSED/*
```

Comment:
It is not a very good idea to list here all possible add-ons directories. Imagine doing this for Doom or Quake with thousands of add-ons.
 Let's leave only directories, related to the original game.

Reply:
Adding the three common main addons to ignore is required for undisturbed in source tree development